### PR TITLE
lesson-5.1

### PR DESCRIPTION
**Pull request by zemiarsi**

### CODE REVIEW

File: lesson_1.1/src/App.js [hunk 1]
```diff
@@ -7,15 +7,18 @@ import SchoolRoundedIcon from '@mui/icons-material/SchoolRounded';
...
 import CTDimg from './img/CTD.png'
+import Toggle from "./Toggle";
 
 
 
 function App() {
   return (
     <BrowserRouter>
-    
+    <div className={style.App}></div>
```
Comment (kaytijackson97) [inline lesson_1.1/src/App.js:17]: fix indentation for return statement

File: lesson_1.1/src/Toggle/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import "./style.css";
+
+
+export default function Toggle() {
+    return (
+    <div className="toggle">
```
Comment (kaytijackson97) [inline lesson_1.1/src/Toggle/index.js:7]: fix indentation for this whole return statement

File: lesson_1.1/src/Toggle/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (kaytijackson97) [inline lesson_1.1/src/Toggle/index.js:1]: semi colon at end of import statement

**Review by kaytijackson97 — COMMENTED** [review]
Indentation is needing a lot of TLC with this section. The return statement of Toggle/index.js really needs so help as its stacked on top of each other and confusing to read.

**Comment by zemiarsi** [comment]
No Toggle is not including in the 5.1 assignment , by forgetting comment out I submitted that is why sorry for the confusion, Thanks!!

**Pull request merged**